scripts/create_github_release.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+SCRIPT_DIR="$(cd "$(dirname "$0")" && pwd)"
+source "${SCRIPT_DIR}/utils.sh"
+
+set -eo pipefail
+
+[[ -n "${GITHUB_TOKEN}" ]] || die "No GITHUB_TOKEN found."
+
+tag="$1"
+bin_dir="$2"
+[[ -n "${tag}" && -n "${bin_dir}" ]] || die "Usage: $0 <tag> <binary_directory>"
+
+mkdir /tmp/release-artifacts
+for os in darwin linux windows; do
+  bin_name="kube-linter"
+  if [[ "${os}" == "windows" ]]; then
+    bin_name="kube-linter.exe"
+  fi
+  tar -C "${bin_dir}/${os}" -czf "/tmp/release-artifacts/kube-linter-${os}.tar.gz" "${bin_name}"
```

Comment:
Why do we package the binary in a tar gz instead of uploading it as-is? Looks like this is common. GH restriction? Security? Purely for preserving permissions?

Reply:
Mostly file size, plus looking at what some other projects do. It was ~30 MB without compression.